plan/new.go

```diff
@@ -52,7 +52,23 @@ func buildTestCases(plan *Plan) []TestCase {
 				Plan:      plan,
 				Client:    client,
 				Arguments: testArgs,
+				Skip:      false,
 			}
+
+			for _, filter := range behavior.SkipFilters {
+				// Each filter should be a complete match
+				isCompleteMatch := true
```

Comment:
A cool way to represent matches in a list like that is to use a counter. `matchCnt := 0`. Increment it at every successful match. After, if `matchCnt == len(filter.AxisMatches)`, you know you got a full match. It has the nice effect if being really easy to read and reason about. But its up to you and mostly a matter of personal taste.